.travis.yml

```diff
@@ -50,16 +49,15 @@ install:
 script:
   - export PIPENV_IGNORE_VIRTUALENVS=1
   # for some reason the migrations check is failing only on django 2.0
-  - if [[ $TRAVIS_PYTHON_VERSION == 3.5 ]]; then pipenv run ./tests/manage.py makemigrations config pki --dry-run | grep "No changes detected"; fi
+  - >
+    pipenv run openwisp-utils-qa-checks
+    --migration-path "./openwisp_controller/config/migrations
+    ./openwisp_controller/connection/migrations
+    ./openwisp_controller/geo/migrations
+    ./openwisp_controller/pki/migrations"
+    --migration-module "config pki"
+    --migrations-to-ignore "12 0 0 4 0"
```

Comment:
Four migrations and 5 migrations to ignore "12 0 0 4 0", is this pushed by mistake?